CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ without native/platform specific bindings and SDKs. See [this ticket for more de
 
 - Drop .NET 6 Mobile in favor of .NET 7 ([#2624](https://github.com/getsentry/sentry-dotnet/pull/2604))
 
+API Changes:
+
+- Removed IHasDistribution ([#2660](https://github.com/getsentry/sentry-dotnet/pull/2660))
```

Comment:
```suggestion
API Changes:

- Adding `Distribution` to `IEventLike` ([#2660](https://github.com/getsentry/sentry-dotnet/pull/2660))
```

Since the removal of the internal interface isn't customer facing, how about this ^